testing/scenariobuilder/builder.go

```diff
@@ -335,7 +336,10 @@ func (s *TodocheckScenario) setupMockIssueTrackerServer() (teardownFunc, error)
 
 		for issue := range s.issues {
 			if r.URL.Path == issuetracker.IssueURLFrom(s.issueTracker, issue) {
-				w.Write(issuetracker.BuildResponseFor(s.issueTracker, issue, s.issues[issue]))
+				_, err := w.Write(issuetracker.BuildResponseFor(s.issueTracker, issue, s.issues[issue]))
+				if err != nil {
+					logger.Info("Error " + err.Error() + " occured, when writing issuetracker response of issue:" + issue)
```

Comment:
This should be a panic as it should never happen

Reply:
I changed the code to panic if this point is reached during execution.